File: programme/area-of-rectangle/area-of-rectangle.java
import java.util.Scanner;

public class area {

	public static void main(String[] args) {
   		    Scanner sc = new Scanner(System.in);
	        System.out.print("Enter the length : ");

	        // taking length input
	        int length = sc.nextInt();

	        System.out.print("\nEnter the breadth : ");

	        //taking breadth input
	        int breadth = sc.nextInt();

	        // calculating area of a rectangle
	        int area =  length * breadth;


	        // printing the final answer
	        System.out.println("Area of rectangle : " + area);

	        sc.close();


	}

}

File: programme/area-of-rectangle/AreaOfRectangle.kt
// Write a programme to compute the area of a rectangle

import java.util.Scanner

class AreaOfRectangle {
    fun main() {
        val sc = Scanner(System.`in`)

        // taking length input
        print("Enter the length  : ")
        val length = sc.nextInt()

        // taking breadth input
        print("Enter the breadth : ")
        val breadth = sc.nextInt()

        // calculating area of a rectangle
        val area = length * breadth

        // printing the final answer
        println("\nArea of rectangle : $area")
        sc.close()
    }
}